### ISSUE TITLE
Error message when member responds with > 160 characters

### ISSUE THREAD

**Issue by jamjensen** [DoSomething/gambit#800]
#### Current Behavior
-  This member triggered a 500 error:"postReportback error:Unexpected end of JSON input" : https://dosomething.slack.com/archives/team-odoyle/p1487951659000039 

- As you can see below, the user received the campaign completion message immediately after receiving the 500 error. 

![image](https://cloud.githubusercontent.com/assets/19572934/23365614/a6ad4c08-fcd1-11e6-9209-b8232a9cf49e.png)

- And after checking their profile, I can confirm that we did receive their reportback -- https://www.dosomething.org/us/user/5300476 

- This might need a separate issue, but in case there is any connection, it should be noted that after successfully reporting back on the campaign they received a relative reminder later that day. 

![image](https://cloud.githubusercontent.com/assets/19572934/23365741/263e00a2-fcd2-11e6-8315-71583e5bdc3c.png)

#### Desired Behavior
- When sending in a message >160 characters, we should recognize it as a single post so that we do not send them an error message or more than one campaign completion message. 

#### Steps to Replicate
- I haven't been able to trigger the 500 error, however, by sending in a `why_participated` > 160 characters I have triggered the `msg_invalid_cmd_completed.` 

- My `why_participated` was split into 4 messages, therefore I received four messages in return: two `campaign_completion` and two `msg_invalid_cmd_completed.`

![image](https://cloud.githubusercontent.com/assets/19572934/23365972/1324d792-fcd3-11e6-92dc-74dba9c57da5.png)

#### Why This Matters
For two reasons: 

1. It's a jarring user experience. We're inundating them with messages (which is annoying) AND it's confusing; the user might not know if they actually completed the campaign or not. 

2. $$$. We're already limited in the number of texts we can send each month. We're throwing away money by sending members redundant messages.

**Comment by aaronschachter** [comment]
The real fix for this is blocked by #725 -- Mobile Commons sends us 2 incoming messages when a single incoming message is longer than 160 chars. If they sent us the entire message, this wouldn't happen.

Expected behavior is that the 2nd half of the Reportback message would trigger a "Sorry I didn't get that" from the Completed Menu -- but there may be a race condition here. 

@jamjensen Would it be worth flagging this for Mobile Commons and checking in on a ETA of a fix? https://github.com/DoSomething/gambit/issues/725#issuecomment-260006147

**Comment by jamjensen** [comment]
@aaronschachter definitely -- will update with their response. 

So, for my message we say expected behavior. How can we determine whether or not a race condition exists? And is it possible that it triggered the 500 error for the user above?

**Comment by jamjensen** [comment]
After MoCo made the switch to Twilio on March 31st, they now send a single post request when the incoming message is > 160 characters.  

**Example of a reportback for SFW: Amplify from 4/4:**

*Screenshot of member's texting history with SFW:Amplify and reason_why >160 characters*
![image](https://cloud.githubusercontent.com/assets/19572934/24682723/32d729b6-1969-11e7-82b1-050bfcc6e375.png)
 
*Screenshot of the member's reportback in Phoenix with a reason_why > 160 characters*
![image](https://cloud.githubusercontent.com/assets/19572934/24682674/f2677476-1968-11e7-8500-67cb45d57a24.png)
